src/main/scala/treadle/utils/AugmentPrintf.scala

```diff
@@ -41,7 +41,8 @@ class AugmentPrintf extends Transform with DependencyAPIMigration {
         case s: Stop =>
           val newStmts = mutable.ArrayBuffer[Statement]()
           val newStop = s.mapExpr(insert(newStmts, namespace, s.info, s.clk))
-          Block(newStmts :+ newStop)
+          newStmts += newStop
```

Comment:
What's the change being addressed by `Block(newStmts :+ newStop)` going to
 ```
newStmts += newPrint
 Block(newStmts.toSeq)
```
And in a previous case you checked the newStmts length and avoided a block. is there a reason for the different pattern here

Reply:
In Scala 2.12, `Seq` is `collection.Seq` (super type of all immutable *and* mutable Seqs). In Scala 2.13, `Seq` is `collection.immutable.Seq`. Thus we cannot pass `mutable.ArrayBuffer` where a `Seq` is required anymore. My change is functionality the same as what was there although I will grant it's a slightly different way to accomplish it. I could've just done:
```scala
Block((newStmts :+ newStop).toSeq)
```
That would've been 2 copies, the `:+` is a copy, followed by conversion to an immutable `Seq`, another copy. Since `newStmts` does not escape this scope, I figured we might as well just mutably append and have the 1 copy. Same number of copies without jumping through any hoops!

An alternative with no copies is to use `mutable.ListBuffer` which has a free `.toList` (it copies if you then mutate the underlying buffer again). The only issue is that `Lists` while decent, aren't great datastructures for memory use. Another alternative is to start using things like `VectorBuilder` in cases like this. I wanted to keep this PR simple so I avoided larger changes like that. My proposal is that we get everything on 2.13 and then switch it to be our default and start measuring performance and improving it.

Reply:
The earlier length check was already there I think, I did the same conversion.